guides/source/testing.md

```diff
@@ -1255,7 +1264,7 @@ end
 Testing Time-Dependent Code
 ---------------------------
 
-Rails provides inbuilt helper methods that enable you to assert that your time-sensitve code works as expected.
+Rails provides inbuilt helper methods that enable you to assert that your time-sensitive code works as expected.
```

Comment:
IMO, "inbuilt" is not proper grammar, and "built-in"/"builtin" is the way to go. This is my first time hearing someone using the word "inbuilt". :(